translations/messages.fr.xlf

```diff
@@ -0,0 +1,3061 @@
+        En fin de partie, la pierre morte est comptée comme une capture, et la case qu&apos;elle occupe comme un territoire.</target>
+      </trans-unit>
+      <trans-unit id="6721138878022657917" datatype="html">
+        <source>Life and death (eyes)</source>
+        <target>Vie et mort (yeux)</target>
+      </trans-unit>
+      <trans-unit id="1084604724991997052" datatype="html">
+        <source>Here, Light cannot play on the top left, nor on the bottom left.
+        Light will never be able to capture Dark.
+        We say that Dark has two eyes (the eye on the top left, and the one on the bottom left) and that Dark is alive.</source>
+        <target>Ici, Clair ne pouvant jouer ni en haut à gauche, ni en bas à gauche, il ne pourra jamais capturer Foncé.
+        On dit alors que Foncé a deux yeux (l&apos;oeil en haut à gauche et celui en bas à gauche) et qu&apos;il est vivant.</target>
+      </trans-unit>
+      <trans-unit id="8745919880228059784" datatype="html">
+        <source>Seki</source>
+        <target>Seki</target>
+      </trans-unit>
+      <trans-unit id="5496499515779223328" datatype="html">
+        <source>If Dark plays on the middle line, Light will play on the opposite intersection and capture Dark.
+        Similarly, if Light plays on the middle line, Dark will capture Light.
+        In other words, there is no point in playing on the middle line.
+        In that case, we say that stones in the middle are alive by Seki, and that both intersections in the middle are neutral.</source>
+        <target>Si Foncé joue au milieu en haut (ou en bas), Clair jouera au milieu en bas (ou en haut) et le capturera.
+        De même, si Clair joue au milieu haut (ou en bas), Foncé le capturera.
+        Autrement dit, personne n&apos;a intérêt à jouer au milieu.
+        Dans ce cas, on dit que les pierres du milieu sont vivantes par Seki, et que les deux intersections du milieu sont des intersections neutres.</target>
+      </trans-unit>
+      <trans-unit id="7812956328094242544" datatype="html">
+        <source>Ko</source>
+        <target>Ko</target>
+      </trans-unit>
+      <trans-unit id="5425125770484596220" datatype="html">
+        <source>A player, by putting a stone, cannot go back to an identical state of the Goban as before, in order to avoid an endless game.&lt;br/&gt;&lt;br/&gt;
+        Capture the light stone.</source>
+        <target>Un joueur, en posant une pierre, ne doit pas redonner au goban un état identique à l&apos;un de ceux qu&apos;il lui avait déjà donné, ce afin d&apos;empêcher qu&apos;une partie soit sans fin.&lt;br/&gt;&lt;br/&gt;
+        Capturez la pierre claire.</target>
+      </trans-unit>
+      <trans-unit id="8268334913018353590" datatype="html">
+        <source>Now, if Light tries to recapture the stone that Dark has put on the board, the Goban would go back to its previous state, opening the door for an endless game.
```

Comment:
Now, if Light tries to recapture the stone that Dark has just put on the Goban, this one would go back to its previous state, [...]